src/librustc_metadata/native_libs.rs

```diff
@@ -162,8 +162,13 @@ impl Collector<'tcx> {
             }
         }
         if lib.cfg.is_some() && !self.tcx.features().link_cfg {
-            feature_err(&self.tcx.sess.parse_sess, sym::link_cfg, span.unwrap(), "is unstable")
-                .emit();
+            feature_err(
+                &self.tcx.sess.parse_sess,
+                sym::link_cfg,
+                span.unwrap(),
+                "kind=\"link_cfg\" is unstable, see https://github.com/rust-lang/rfcs/pull/1721",
```

Comment:
I don't think we need to mention https://github.com/rust-lang/rfcs/pull/1721 here — we already mention the relevant issue below. However, https://github.com/rust-lang/rust/issues/37406 seems to be closed, so maybe we'll need to open it.
```suggestion
                "kind=\"link_cfg\" is unstable",
```

Reply:
Removed.